schema-admin.graphql

```diff
@@ -862,6 +862,16 @@ type Query {
         url: String!
     ): ApprovedCorpusItem
 
+    """
+    Retrieves an approved item with the given external ID.
+    """
+    approvedCorpusItem(
```

Comment:
```suggestion
    approvedCorpusItemByExternalId(
```

Reply:
Done! We can use this and keep `approvedCorpusItem` available for when `@oneof` directive is available as discussed at today's pairing: https://github.com/graphql/graphql-spec/pull/825#issuecomment-938051965